app/backend/lib/emails/templates/assesmentSecondReviewChange.ts

```diff
@@ -3,21 +3,33 @@ import {
   EmailTemplateProvider,
 } from '../handleEmailNotification';
 
+const formats = {
+  projectManagement: { type: 'Project Management', slug: 'project-management' },
+  permitting: { type: 'Permitting', slug: 'permitting' },
+  technical: { type: 'Technical', slug: 'technical' },
+  gis: { type: 'GIS', slug: 'gis' },
+  financialRisk: { type: 'Financial Risk', slug: 'financial-risk' },
+  screening: { type: 'Screening', slug: 'screening' },
+};
+
 const assesmentSecondReviewChange: EmailTemplateProvider = (
   applicationId: string,
   url: string,
   initiator: string,
   params: any
 ): EmailTemplate => {
-  const { ccbcNumber } = params;
+  const { ccbcNumber, assessmentType } = params;
+  const { type, slug } = formats[assessmentType];
+
   return {
     emailTo: [34, 71], // Temporary IDs to handle email recipients
     emailCC: [],
     tag: 'assesment-second-review-change',
-    subject: `${initiator} requested a 2nd Review for Eligibility Screening - ${ccbcNumber}`,
+    subject: `${initiator} has requested a 2nd Review for Eligibility ${type} - ${ccbcNumber}`,
     body: `
-    <h1>${initiator} requested a 2nd Review for Eligibility Screening - ${ccbcNumber}</h1>
-    <p>${initiator} requested a 2nd Review for Eligibility Screening - ${ccbcNumber}, <a href='${url}/analyst/application/${applicationId}/assessments/screening'>Click here</a> to view the Eligibility Screening in the CCBC Portal<p>
+    <h1>${initiator} requested a 2nd Review on Eligibility ${type} - ${ccbcNumber}</h1>
+    <p>${initiator} requested a 2nd Review on Eligibility ${type} - ${ccbcNumber}, <a href='${url}/analyst/application/${applicationId}/assessments/${slug}'>Click here</a> to view the Eligibility ${type} in the CCBC Portal<p>
```

Comment:
I think the "Eligibility" here isn't part of the ticket? Might need some clarification from Mars but I believe screening is the only one we include "Eligibility" here.

Reply:
@AntBush Changed to reflect word 'assessment', please give another look whenever you can